geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/SimpleFeatureConverterFactory.scala

```diff
@@ -145,8 +189,10 @@ trait ToSimpleFeatureConverter[I] extends SimpleFeatureConverter[I] with LazyLog
         ec.set(i, requiredFields(i).eval(t)(ec))
       } catch {
         case e: Exception =>
-          logger.warn(s"Failed to evaluate field '${requiredFields(i).name}' using values:\n" +
+          Try{
```

Comment:
we really shouldn't have trys around our logger statements... can we re-write the logger string so that it handles whatever the error was? (NPE, OutOfBounds, etc)